website/templates/login.html

```diff
@@ -0,0 +1,27 @@
+    {% extends "base.html" %}
+    {% block pageContent%}
+    <body>
+    <h1>Welcome to ZoomFront!</h1>
+    {% if session['username'] %}
+    <div class="container">
+        <p> Oh. This is embarassing. You're not supposed to be here when youre logged in, yknow? </p>
```

Comment:
Hehe. Isn't the session variable storing the logged in user named `session['userID']` or `session['viewerID']`?